File: Implementation/backend/src/main/java/ch/heiafr/arsi/g6/hashcode/constant/Roles.java
package ch.heiafr.arsi.g6.hashcode.constant;

import ch.heiafr.arsi.g6.hashcode.model.Role;

public abstract class Roles {
  public static final Role ADMIN = new Role(1, "ROLE_ADMIN");
  public static final Role VALIDATED_ORGANIZER = new Role(2, "ROLE_VALIDATED_ORGANIZER");
  public static final Role PENDING_ORGANIZER = new Role(3, "ROLE_PENDING_ORGANIZER");
  public static final Role VALIDATED_USER = new Role(4, "ROLE_VALIDATED_USER");
  public static final Role PENDING_USER = new Role(5, "ROLE_PENDING_USER");
}

File: Implementation/frontend/src/app/constant/roles.ts
import {Role} from '../model/Role';

export abstract class Roles {
  public static readonly ADMIN: Role = new Role(1, 'ROLE_ADMIN');
  public static readonly VALIDATED_ORGANIZER: Role = new Role(2, 'ROLE_VALIDATED_ORGANIZER');
  public static readonly PENDING_ORGANIZER: Role = new Role(3, 'ROLE_PENDING_ORGANIZER');
  public static readonly VALIDATED_USER: Role = new Role(4, 'ROLE_VALIDATED_USER');
  public static readonly PENDING_USER: Role = new Role(5, 'ROLE_PENDING_USER');
}
